applications/ContactStructuralMechanicsApplication/symbolic_generation/mesh_tying_mortar_condition/custom_sympy_fe_utilities.py

```diff
@@ -1,492 +0,0 @@
-from sympy import *
```

Comment:
are u removing this file?

Reply:
I am moving the file

Reply:
remove it. You should be using the one in the core. What are the differences?